### PR TITLE
Something that works: Rough draft

### PR DESCRIPTION
**Pull request by aanil**

### CODE REVIEW

File: daily_read/templates/daily_report.html [hunk 1]
```diff
@@ -113,39 +113,56 @@ <h2><small>
...
+                    </div>
+                    <div class="progress-bar progress-bar-success" role="progressbar" aria-valuenow="2" aria-valuemin="0" aria-valuemax="5" style="width: 16.67%">
+                      All Raw Data delivered
+                    </div>
+                    <div class="progress-bar progress-bar-success" role="progressbar" aria-valuenow="2" aria-valuemin="0" aria-valuemax="5" style="width: 16.65%">
+                      Project closed
+                    </div>
+                  </div>
+                  {% set vars = {'h2_set': false }%}
+                  {% for date_prio in priority %}
```
Comment (aanil) [inline daily_read/templates/daily_report.html:157]: Try dictsort

File: daily_read/__main__.py [hunk 2]
```diff
@@ -37,16 +38,34 @@ def generate():
...
     op.get_orders()
-    # TODO - not sure how to proceed, fetching ALL projects in order portal 
-    # and then generate reports seems quite slow and wasteful, but I haven't 
-    # tried so it might work. Maybe a pull model where we only generate reports 
+    # TODO - not sure how to proceed, fetching ALL projects in order portal
+    # and then generate reports seems quite slow and wasteful, but I haven't
+    # tried so it might work. Maybe a pull model where we only generate reports
     # for projects that we know have been updated can be a better alternative?
+    all_sthlm_orders = op.process_orders(use_node='Stockholm')
+    daily_rep = daily_read.daily_report.DailyReport(config_values.DAILY_READ_REPORTS_LOCATION)
```
Comment (aanil) [inline daily_read/__main__.py:50]: Should clean up

File: daily_read/__main__.py [hunk 3]
```diff
@@ -37,16 +38,34 @@ def generate():
...
+            op.upload_report_to_order_portal(report, project['iuid'])
+
 
 @generate.command(name='single')
 @click.argument('orderer')
-@click.argument('location')
-def generate_single():
-    pass
+@click.option('-l', '--location', type=click.Choice(['Stockholm', 'Uppsala'], case_sensitive=False))
+def generate_single(orderer, location):
```
Comment (aanil) [inline daily_read/__main__.py:69]: Orderer and location together?

File: daily_read/templates/daily_report.html [hunk 4]
```diff
@@ -113,39 +113,56 @@ <h2><small>
...
-                                <h2 class="panel-title">Project B <span class="caret"></span></h2>
-                            </div>
-                            <div class="col-md-3 text-right">Last updated: 2022-04-15</div>
-                        </div>
-                    </a>
-                </div>
-                <div class="panel-body collapse" id="panel_project_b">
-                    Panel content
+                <div class="panel-body collapse" id="panel_project_{{ project['project_id'] }}">
+                  <div class="progress">
```
Comment (aanil) [inline daily_read/templates/daily_report.html:135]: Rewrite progress bars and make text bigger too

File: daily_read/__main__.py (hunk 3 above)
Comment (alneberg) [reply daily_read/__main__.py:69]:
```suggestion
#TODO: is it possible to filter on orderer and location together in the API?
def generate_single(orderer, location):
```

File: daily_read/__main__.py (hunk 2 above)
Comment (alneberg) [reply daily_read/__main__.py:50]:
```suggestion
    #TODO: Should clean up...
    daily_rep = daily_read.daily_report.DailyReport(config_values.DAILY_READ_REPORTS_LOCATION)
```

File: daily_read/templates/daily_report.html (hunk 4 above)
Comment (alneberg) [reply daily_read/templates/daily_report.html:135]:
```suggestion
                  <!-- TODO: Rewrite progress bars and make text bigger too-->
                  <div class="progress">
```

File: daily_read/templates/daily_report.html (hunk 1 above)
Comment (alneberg) [reply daily_read/templates/daily_report.html:157]:
```suggestion
                  #TODO: try sorting the dict
                  {% for date_prio in priority %}
```

**Pull request merged**